src/ADAL.PCL/TokenCache.cs

```diff
@@ -163,7 +163,7 @@ public byte[] Serialize()
                     writer.Write(this.tokenCacheDictionary.Count);
                     foreach (KeyValuePair<TokenCacheKey, AuthenticationResultEx> kvp in this.tokenCacheDictionary)
                     {
-                        writer.Write(string.Format(CultureInfo.CurrentCulture, "{1}{0}{2}{0}{3}{0}{4}", Delimiter,
```

Comment:
Although not *technically* an issue (since we're only using this as a simple string-concat and not invoking any formating options), changing this to use the correct culture (the invariant one).
What we lay down on disk should not depend on the current culture as this might change between serialisation and deserialisation.